IPython/html/static/base/less/page.less

```diff
@@ -5,16 +5,23 @@
  */
 
 
+html, body {
+    min-height: 100%;
+    height: 100%;
+    overflow: hidden; // Prevents scrollbar in html element
```

Comment:
This is necessary. `overflow: hidden;` prevents the creation of a temporary scrollbar in the html element after loading the page . `min-height: 100%; height: 100%;` sometimes is added to CSS resets and more frequently in fixed header layouts.